cmd/cca/check_test.go

```diff
@@ -192,3 +192,26 @@ func Test_CheckCmd_token_invalid_format(t *testing.T) {
 	err = cmd.Execute()
 	assert.EqualError(t, err, expectedErr)
 }
+
+func Test_CheckCmd_regression_bug_18(t *testing.T) {
+	fs := afero.NewMemMapFs()
+
+	err := afero.WriteFile(fs, "ccatoken.cbor", testValidCCAToken, 0644)
+	require.NoError(t, err)
+
+	err = afero.WriteFile(fs, "es256.jwk", testValidPAK, 0644)
+	require.NoError(t, err)
+
+	cmd := NewCheckCmd(fs)
+	cmd.SetArgs(
+		[]string{
+			"--token=ccatoken.cbor",
+			"--key=es256.jwk",
```

Comment:
--key should be 384 right?

Reply:
For CCA?

Reply:
it really doesn't matter here, it could be called any name

Reply:
I've given it a more eloquent name